source-map.bs

```diff
@@ -65,7 +65,7 @@ spec:url; type:dfn; for:/; text:url
     "title": "Source Map Revision 2 Proposal"
   },
   "WasmCustomSection": {
-    "href": "https://www.w3.org/TR/wasm-core-2/binary/modules.html#custom-section",
+    "href": "https://www.w3.org/TR/wasm-core-2/#custom-section%E2%91%A0",
```

Comment:
Shouldn't it be just `https://www.w3.org/TR/wasm-core-2/#custom-section`?

Reply:
Should we link to the https://www.w3.org/TR/wasm-core-2/#custom-section or the https://webassembly.github.io/spec/core/binary/modules.html#custom-section

Reply:
There is no difference in content. I usually check the second variant of the specification (because of the less contrast colors). The only think came into my mind, that the `w3.org` version looks more official and from the official specification it would be better to mention other official specification. 
But what do you think fit better?

Reply:
That makes sense to me, let's go with the w3.org one because it does look nice and official!